### PR TITLE
21.0.3

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 21.0.3

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: conf/config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "system": {
     "updatechecker": false,
-    "memcache.local": "\\OC\\Memcache\\APCu",
+    "memcache.local": "\\OC\\Memcache\\Redis",
```
Comment (kay0u) [inline conf/config.json:4]: Hmm why did you change this value?

Comment (ericgaspar) [reply conf/config.json:4]: i had a failed install with a warning about something involving APCu (sorry I can't recall the specific warning).
I am reloading a test with APCu set

**Comment by ericgaspar** [comment]
!testme

File: conf/config.json (hunk 1 above)
Comment (kay0u) [reply conf/config.json]: It could be related to https://github.com/YunoHost-Apps/nextcloud_ynh/issues/419

I have to admit that I really don't know what is the best option here.

Comment (ericgaspar) [reply conf/config.json]: I found a topic on the subject on Nextcloud forum. Trying to find it...

Comment (ericgaspar) [reply conf/config.json]: The CI warning:
`141712 INFO DEBUG - OC\HintException: [0]: Memcache \OC\Memcache\APCu not available for local cache (Is the matching PHP module installed and enabled?)`

The issue I found on Nextcloud forum:
https://help.nextcloud.com/t/solved-apcu-caching-gives-error/9403/5

but to be fair I don't understand either what the implications are... `¯\_ (ツ) _/¯`

Comment (kay0u) [reply conf/config.json]: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/caching_configuration.html#id1

We need to install a package, and enable it in the [php conf file](https://github.com/YunoHost-Apps/nextcloud_ynh/blob/testing/conf/extra_php-fpm.conf)

so it's related to #419 I'll fix that later (how it worked before O.O)

Comment (ericgaspar) [reply conf/config.json]: Will it make sense to only use Redis for memory caching?  it seems now that we are using APCu and Redis (cf. config.json)

Comment (kay0u) [reply conf/config.json]: According to the doc: 

> APCu is faster at local caching than Redis. If you have enough memory, use APCu for Memory Caching and Redis for File Locking. If you are low on memory, use Redis for both.

**Comment by kay0u** [comment]
!testme

File: conf/config.json (hunk 1 above)
Comment (kay0u) [reply conf/config.json]: It's not a bug, it's a feature:

https://github.com/nextcloud/server/issues/27781#issuecomment-873543420

**Comment by kay0u** [comment]
!testme

**Comment by nathanael-h** [comment]
Hello, as the test is OK could this PR be merged, at least in testing ? I saw that there is a security warning a few days ago : EN  https://github.com/nextcloud/security-advisories/security/advisories/GHSA-6qr9-c846-j8mg ;  FR  https://www.cert.ssi.gouv.fr/avis/CERTFR-2021-AVI-543/ 
If I can help (testing..?) ...

**Comment by nathanael-h** [comment]
Hello, I updated a test instance on my server, from 21.0.2 to 21.0.3 it is ok. The cron seems to run well. 
Ping @ericgaspar and @kay0u 😃

**Review by kay0u — APPROVED** [review]

**Comment by kay0u** [comment]
it ok for me
